packages/telemetry/src/api.ts

```diff
@@ -44,3 +52,49 @@ export function getTelemetry(app: FirebaseApp = getApp()): Telemetry {
 
   return telemetryProvider.getImmediate();
 }
+
+/**
+ * Enqueues an error to be uploaded to the Firebase Telemetry API.
+ *
+ * @public
+ *
+ * @param telemetry - The {@link Telemetry} instance.
+ * @param error - the caught exception, typically an {@link Error}
+ */
+export function captureError(telemetry: Telemetry, error: unknown): void {
+  const logger = telemetry.loggerProvider.getLogger('error-logger');
+  if (error instanceof Error) {
+    logger.emit({
```

Comment:
Just out of curiosity: this makes a network call using the opentelemetry library's LoggerProvider?

Reply:
That's correct